core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/jdbc/JDBCUserStoreManager.java

```diff
@@ -4594,6 +4627,12 @@ private void buildClaimWhereConditions(SqlBuilder sqlBuilder, String attributeNa
             sqlBuilder.where("UA.UM_ATTR_VALUE LIKE ?", "%" + attributeValue + "%");
         } else if (ExpressionOperation.SW.toString().equals(operation)) {
             sqlBuilder.where("UA.UM_ATTR_VALUE LIKE ?", attributeValue + "%");
+        } else if (ExpressionOperation.GT.toString().equals(operation)) {
+            sqlBuilder.where("UA.UM_ATTR_VALUE > ?", attributeValue);
+        } else if (ExpressionOperation.LT.toString().equals(operation)) {
+            sqlBuilder.where("UA.UM_ATTR_VALUE < ?", attributeValue);
+        } else if (ExpressionOperation.NE.toString().equals(operation)) {
+            sqlBuilder.where("UA.UM_ATTR_VALUE != ?", attributeValue);
```

Comment:
sqlBuilder.where("UA.UM_USER_ID NOT IN (SELECT UM_USER_ATTRIBUTE.UM_USER_ID FROM UM_USER_ATTRIBUTE WHERE UM_USER_ATTRIBUTE.UM_ATTR_VALUE == ?)", attributeValue);